acq4/util/micromanager.py

```diff
@@ -2,16 +2,47 @@
 from __future__ import print_function
 import os, sys
 from acq4.util.Mutex import Mutex
-
+from acq4 import CONFIGPATH
+from acq4.util import configfile
 # singleton MMCorePy instance
+
+
+
 _mmc = None
+global _mmc
+
+
+DEFAULT_MM_PATH = "C:\\Program Files\\Micro-Manager-1.4"
+
+def getMMConfig():
+    """
+        get the micromanager path and desired micromanager config file from acq4 config 
+    """    
+    for path in CONFIGPATH:
+        cf = os.path.join(path, 'default.cfg')
+        if os.path.isfile(cf):
+            cfg = configfile.readConfigFile(cf)
```

Comment:
Rather than re-reading the config file, maybe the Manager should have a method for accessing this, like `getConfig()` ?